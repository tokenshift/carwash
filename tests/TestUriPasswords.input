postgres://dbuser:dbpassword@dbhost:5432/somedb
Here's a log entry: postgres://username:secretd@dbhost:5432/somedb and some more text: jdbc:postgresql://foo:bar@db.example.com/fizzbuzz the end.
postgres://db.example.com/database
postgres://dbuser:password%21@dbhost:5432/somedb
This is the password! from above.
