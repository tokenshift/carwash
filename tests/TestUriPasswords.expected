postgres://dbuser:********@dbhost:5432/somedb
Here's a log entry: postgres://username:********@dbhost:5432/somedb and some more text: jdbc:postgresql://foo:********@db.example.com/fizzbuzz the end.
postgres://db.example.com/database
postgres://dbuser:********@dbhost:5432/somedb
This is the ******** from above.
